quantus/metrics/localisation/attribution_localisation.py

```diff
@@ -243,9 +242,7 @@ def __call__(
 
     def evaluate_instance(
```

Comment:
I really like how you have removed unnecessary args through your reformulation, great work!